notes.md

```diff
@@ -0,0 +1,13 @@
+    //css: 
+    //centering
+    //to fill whole page
+    //buttons interaction
+    //buttons in mobile screen
+
+    // articles:
+    //get rid of page 0
+    //set maximum pages
+
+    //try to refactor queries as in Dave's lections
+    //also look at class usage
+    //error handling
```

Comment:
Try not to commit files like this. As a tip add it to your gitignore so it doesn't happen by accident.

Reply:
This file is a part of on-the-way-planning, there go comments from code about changes that I have not done before pull request but going to do later. Added to gitignore, it does not make much sense, it is not collecting updates and is discarded every time brunch is deleted. I can store it on my desktop, but just thought it's better to keep it with the project while production phase and discard it before the deployment of the app. Is it not allowed?